frontend/express/public/core/user-management/javascripts/countly.views.js

```diff
@@ -386,9 +394,66 @@
                 this.permissionSets.splice(index, 1);
                 this.$set(this.$refs.userDrawer.editedObject.permission._.u, this.$refs.userDrawer.editedObject.permission._.u.splice(index, 1));
             },
+            setPermissionByDependency: function(index, type, feature) {
```

Comment:
Does this belong in the views.js or can we turn this into an action in the models file?

Reply:
all the similar logic ex: automatic read permission, also exists in view.js hence I decided to keep similar things together 😀

Reply:
Okay 👍
We should refactor this once the changes are tested and stable
FYI @widatama